Sources/TSCBasic/FileSystem.swift

```diff
@@ -343,6 +349,10 @@ private class LocalFileSystem: FileSystem {
         try FileManager.default.createDirectory(atPath: path.pathString, withIntermediateDirectories: recursive, attributes: [:])
     }
 
+    func createSymbolicLink(_ path: AbsolutePath, pointingAt destination: AbsolutePath) throws {
+        try FileManager.default.createSymbolicLink(atPath: path.pathString, withDestinationPath: destination.pathString)
```

Comment:
Commented on this below, but I think the relative form of this is still important.  The code from the original method:
```
    let destString = relative ? dest.relative(to: path.parentDirectory).pathString : dest.pathString
```
still seems relevant here.